content/reference-transaction-format.md

```diff
@@ -1312,7 +1312,12 @@ These codes indicate an error in the local server processing the transaction; it
+| `telCAN_NOT_QUEUE_BALANCE` | The transaction did not meet the [open ledger cost](concept-transaction-cost.html) and also was not added to the transaction queue because the sum of potential XRP costs of already-queued transactions is greater than the expected balance of the account. You can try again later, or try submitting to a different server. |
+| `telCAN_NOT_QUEUE_BLOCKS` | The transaction did not meet the [open ledger cost](concept-transaction-cost.html) and also was not added to the transaction queue. This transaction could not replace an existing transaction in the queue because it would block already-queued transactions from the same sender by changing authorization methods. (This includes all [SetRegularKey][] and [SignerListSet][] transactions, as well as [AccountSet][] transactions that change the RequireAuth/OptionalAuth, DisableMaster, or AccountTxnID flags.) You can try again later, or try submitting to a different server. |
+| `telCAN_NOT_QUEUE_BLOCKED` | The transaction did not meet the [open ledger cost](concept-transaction-cost.html) and also was not added to the transaction queue because a transaction queued ahead of it from the same sender blocks it. (This includes all [SetRegularKey][] and [SignerListSet][] transactions, as well as [AccountSet][] transactions that change the RequireAuth/OptionalAuth, DisableMaster, or AccountTxnID flags.) You can try again later, or try submitting to a different server. |
+| `telCAN_NOT_QUEUE_FEE` | The transaction did not meet the [open ledger cost](concept-transaction-cost.html) and also was not added to the transaction queue because a transaction with the same sender and sequence number already exists in the queue and this one does not pay a large enough transaction cost to replace it. To replace a transaction in the queue, you the new transaction must have a `Fee` value that is at least 25% higher in [fee levels](concept-transaction-cost.html#fee-levels). You can increase the `Fee` and try again, send this with a higher `Sequence` number so it doesn't replace an existing transaction, or try sending to another server. |
+| `telCAN_NOT_QUEUE_FULL` | The transaction did not meet the [open ledger cost](concept-transaction-cost.html) and the server did not queue this transaction because this server's transaction queue is full. You could increase the `Fee` and try again, try again later, or try submitting to a different server. To replace a transaction in the queue, the new transaction must have `Fee` value that is at least 25% higher in [fee levels](concept-transaction-cost.html#fee-levels) than the transaction it replaces. |
```

Comment:
> To replace a transaction in the queue, the new transaction must have `Fee` value that is at least 25% higher in [fee levels](concept-transaction-cost.html#fee-levels) than the transaction it replaces.

This is not true for the `_FULL` case. All the new transaction needs is to have a higher fee than the cheapest transaction in the queue.